### PR TITLE
Add block for checkout footer copyright_link

### PR DESCRIPTION
**Pull request by jdaandersj**
Put the checkout copyright link into a block for easier child theme workflow.

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Putting the copyright link on the checkout templates footer into a block to enable easier child theme override (matching the main store template footer)
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | n/a
| How to test?      | Install theme, rebuild from templates, ensure checkout footer is not broken. Extra testing if desired: verify using the block in a child theme works.
| Possible impacts? | Should only impact the footer on 'checkout' pages

### CODE REVIEW

**Review by SharakPL — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]
Makes sense but not sure if we can push it to QA without an issue, wdyt @PrestaShop/prestashop-maintainers ?

**Comment by kpodemski** [comment]
@NeOMakinG I think so, it's pretty straightforward to test the PR

**Review by kpodemski — APPROVED** [review]

**Comment by AureRita** [comment]
Hi @jdaandersj 

You can create a feature request[ just here, 
](https://github.com/PrestaShop/PrestaShop/issues/new?assignees=&labels=Feature%2CNew&template=2_feature_request.yml)
PM can validate it 

Thank you,

**Comment by Hlavtox** [comment]
@AureRita This does not need PM approval, it's fixing a missing block, which is present in the other footer - https://github.com/PrestaShop/classic-theme/blob/develop/templates/_partials/footer.tpl.

**Review by MhiriFaten — APPROVED** [review]
Hello @jdaandersj ,

It's QA approved :heavy_check_mark: 

Thank you.

**Comment by Hlavtox** [comment]
Thanks @jdaandersj!

**Pull request merged**
